### PR TITLE
feat: Add Missing Tab Strings and Previously Unimplemented Tab Strings

### PR DESCRIPTION
**Pull request by matthe815**
This implements tab strings that were missed by my previous PR as well as an attempt to add tab titles for pages which should've had them but do not, e.g: Gearsets, Crafting Rotations.

### CODE REVIEW

File: apps/client/src/app/pages/simulator/components/rotation-folder-page/rotation-folder-page.component.ts [hunk 1]
```diff
@@ -17,6 +17,9 @@ import { FlexModule } from '@angular/flex-layout/flex';
 import { UserAvatarComponent } from '../../../../modules/user-avatar/user-avatar/user-avatar.component';
 import { NzCardModule } from 'ng-zorro-antd/card';
 import { NgIf, NgFor, AsyncPipe } from '@angular/common';
+import { SeoMetaConfig } from '../../../../core/seo/seo-meta-config';
```
Comment (Supamiu) [inline apps/client/src/app/pages/simulator/components/rotation-folder-page/rotation-folder-page.component.ts:20]: I don't think this import is needed?

Comment (matthe815) [reply apps/client/src/app/pages/simulator/components/rotation-folder-page/rotation-folder-page.component.ts:20]: Oh I missed  that. Thank you.

File: apps/client/src/app/pages/simulator/components/rotation-folder-page/rotation-folder-page.component.ts [hunk 2]
```diff
@@ -54,19 +58,24 @@ export class RotationFolderPageComponent extends SeoPageComponent {
...
+      }),
+      takeUntilDestroyed(this.destroyRef)
+    ).subscribe(meta => {
+      this.seoService.setConfig(meta)
+    })
+  }
+
+  protected getSeoMeta(): Observable<Partial<SeoMetaConfig>> {
+    return
   }
```
Comment (Supamiu) [inline apps/client/src/app/pages/simulator/components/rotation-folder-page/rotation-folder-page.component.ts]: I don't think this is needed ^^

**Review by Supamiu — APPROVED** [review]

File: apps/client/src/app/pages/simulator/components/rotation-folder-page/rotation-folder-page.component.ts (hunk 2 above)
Comment (matthe815) [reply apps/client/src/app/pages/simulator/components/rotation-folder-page/rotation-folder-page.component.ts]: Oops

Comment (matthe815) [reply apps/client/src/app/pages/simulator/components/rotation-folder-page/rotation-folder-page.component.ts]: Oh that file appears to be outdated, I fixed that already.

**Pull request merged**